hack/arktos-cni.rc

```diff
@@ -161,7 +161,15 @@ install_arktos_containerd() {
 
 install_flannel() {
     local networkcidr="${1:-10.244.0.0/16}"
-    local apiserver_ip=${2:-127.0.0.1}
+    local apiserver_ip=${API_HOST_IP_EXTERNAL:-127.0.0.1}
+
+    # hack/arktos-worker-up.sh does not call hack/lib/common-var-init.sh, which defines the parameter ${lohostname}
```

Comment:
good catch of this discrepancy. Is it possible to change arktos-worker-up to compensate this, instead of at the here?

Reply:
In hack/arktos-worker-up.sh, if the script hack/lib/common-var-init.sh is included, the new codes here are not necessary. I will remove these new codes here.